src/renderer/ipc.ts

```diff
@@ -24,12 +24,14 @@ function getImplementation(): IPCInterface {
   // https://github.com/getsentry/sentry-javascript/blob/a3f70d8869121183bec037571a3ee78efaf26b0b/packages/browser/src/integrations/breadcrumbs.ts#L240
   return {
     sendScope: (scope) => {
-      fetch(`${PROTOCOL_SCHEME}://${IPCChannel.SCOPE}/sentry_key`, { method: 'POST', body: scope }).catch(() => {
+      const init: RequestInit = { mode: 'cors', method: 'POST', body: scope };
+      fetch(`${PROTOCOL_SCHEME}://${IPCChannel.SCOPE}/sentry_key`, init).catch(() => {
         // ignore
       });
     },
     sendEvent: (event) => {
-      fetch(`${PROTOCOL_SCHEME}://${IPCChannel.EVENT}/sentry_key`, { method: 'POST', body: event }).catch(() => {
+      const init: RequestInit = { mode: 'cors', method: 'POST', body: event };
+      fetch(`${PROTOCOL_SCHEME}://${IPCChannel.EVENT}/sentry_key`, init).catch(() => {
```

Comment:
ah the `sentry_key` hack lol. The team is still undecided on how to best resolve that, so it'll stay like this for a while.

As an aside point, we should probably have a quick e2e test to regression test against adding ipc fetch breadcrumbs - just in case we mess something up in the main JS SDK. We should do it outside of this PR though.

Reply:
I'll give it a test but I seem to remember that without this, most of the tests fail because we end up in an infinite loop creating breadcrumbs and sending scope that results in more breadcrumbs!